### ISSUE TITLE
Determine color to fix contrast issue in header

### ISSUE THREAD

**Issue by caw310** [department-of-veterans-affairs/vets-design-system-documentation#5943]
## Description
Review the report from MHV and determine the color that should be used to replace the blue that is bleedingi not the header. 
[[Accessibility]: Current section indicator with insufficient contrast](https://github.com/department-of-veterans-affairs/vets-design-system-documentation/issues/5942)

## Details

<img width="448" height="228" alt="Image" src="https://github.com/user-attachments/assets/029c29c1-8ee5-428b-be10-51e8d520dce3" />

## Tasks
- [ ] Review the issue that was reported
- [ ] Identify what color should be used instead
- [ ] Get a review by designer and accessibility specialist
- [ ] Add the recommended color to be used in issue #5942 

## Acceptance Criteria
- [ ] Color recommendation is in issue 5942
